vendordeps/Phoenix.json

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
This dependency is already here under a different name, don't need it again

Reply:
I'll delete it